cmd/ingress/ingress.go

```diff
@@ -192,6 +192,7 @@ For example, no available LB exists in the bare metal environment.`)
 	cmd.PersistentFlags().StringVar(&cfg.APISIX.DefaultClusterName, "default-apisix-cluster-name", "default", "name of the default apisix cluster")
 	cmd.PersistentFlags().DurationVar(&cfg.ApisixResourceSyncInterval.Duration, "apisix-resource-sync-interval", 300*time.Second, "interval between syncs in seconds. Default value is 300s.")
 	cmd.PersistentFlags().StringVar(&cfg.PluginMetadataConfigMap, "plugin-metadata-cm", "plugin-metadata-config-map", "ConfigMap name of plugin metadata.")
+	cmd.PersistentFlags().BoolVar(&cfg.DisableStatus, "disable-status", false, "Disable resource status updates")
```

Comment:
disable-status-updates ?

Reply:
SGTM!

Reply:
`disable-crd-status`? I think the "disable" contains "disable update".

Reply:
The `status` here will also include resources such as Ingress and Gateway API

Reply:
> The `status` here will also include resources such as Ingress and Gateway API

Yeah. PR contains ingress.

Reply:
>  I think the "disable" contains "disable update".

@tao12345666333 What about you?

Reply:
> I think the "disable" contains "disable update".

"disable" is a relatively vague meaning. For example, "disable status update", "disable status display" or "disable status cache" 

I think that as a configuration item, it should be as clear as possible, so that users can understand its function when using it, and avoid ambiguity